README.md

```diff
@@ -1,99 +1,51 @@
-    Lit components using `@lit-labs/ssr` with optional hydration.
-  - [`@lit-labs/react`](./packages/labs/react) - A React component wrapper for web components.
-  - [`@lit-labs/router`](./packages/labs/router) - A router for Lit.
-  - [`@lit-labs/task`](./packages/labs/task) - A controller for Lit that renders asynchronous tasks.
-  - [`@lit-labs/motion`](./packages/labs/motion) - Lit directives for making things move
-  - [`@lit-labs/scoped-registry-mixin`](./packages/labs/scoped-registry-mixin) - A mixin for LitElement that integrates with the speculative Scoped CustomElementRegistry polyfill.
-- Starter kits (not published to npm)
-  - [`lit-starter-ts`](./packages/lit-starter-ts) ([template
-    repo](https://github.com/lit/lit-element-starter-ts)) - A starter repo for building reusable components using Lit in TypeScript.
-  - [`lit-starter-js`](./packages/lit-starter-js) ([template
-    repo](https://github.com/lit/lit-element-starter-js)) - A starter repo for building reusable components using Lit in Javascript.
-- Internal packages (not published to npm)
-  - [`tests`](./packages/tests) - Test infrastructure for the monorepo.
-  - [`benchmarks`](./packages/benchmarks) - Benchmarks for testing various libraries in the monorepo.
-  - [`@lit-internal/scripts`](./packages/@lit-internal/scripts) - Utility scripts used within the monorepo.
-
-## Contributing to Lit
-
-Lit is open source and we appreciate issue reports and pull requests. See [CONTRIBUTING.md](./CONTRIBUTING.md) for more information.
-
-### Setting up the lit monorepo for development
-
-Initialize repo:
+| Original name         | Fork name                   |
+|-----------------------|-----------------------------|
+| lit                   |  @gynzy/lit                 |
+| lit-element           |  @gynzy/lit-element         |
+| lit-html              |  @gynzy/lit-html            |
+| @lit/reactive-element | @gynzy/lit-reactive-element |
 
-```sh
-git clone https://github.com/lit/lit.git
-cd lit
-npm ci
-```
+### Scoped dependencies
 
-Build all packages:
+Some packages depend on each other (e.g. `lit-element` depends on `@lit/reactive-element` and `lit-html`). This fork overrides these dependencies with versions from the fork.
 
-```sh
-npm run build
-```
-
-Test all packages:
+## Releasing a new version
 
-```sh
-npm run test
-```
+### Automatically
 
-Run benchmarks for all packages:
+At the moment no CI exists. For now, the manual steps can be used.
 
-```sh
-npm run benchmarks
-```
+### Manually
 
-See individual package READMEs for details on developing for a specific package.
+1. Use node 16.x.
+2. Checkout the tag of the latest release.
+3. Install all packages using `npm ci`.
+4. Ensure the required patches are applied / have no conflict.
+5. Build all packages using `npm run build`.
+6. Publish the packages to the private npm repository: `npm publish`. The `@gynzy`-scope ensures that the package is published to the private Gynzy repository.
```

Comment:
@Frank3K Can you do all npm commands, especially npm publish, in the main directory or should you do that in packages/lit ?

Reply:
Good one. Steps 1-5 should be run in the main directory. Step 6 is per-package.